app/components/signup-form.js

```diff
@@ -25,19 +22,14 @@ export default Component.extend({
     return this.get('user.password') || '';
   }),
 
-  init() {
-    this._super(...arguments);
-    this.set('user', this.get('store').createRecord('user'));
```

Comment:
Moved this to the route so we don't have to inject the store in the component anymore.